app/models/cda/prosecution_case_search.rb

```diff
@@ -0,0 +1,10 @@
+module Cda
+  class ProsecutionCaseSearch < BaseModel
+    self.element_name = "prosecution_case"
+
+    def self.execute(filter_params)
```

Comment:
I think we use `.call` instead of execute

Reply:
I've tried to fix your comment about "it is a model, but it behaves like a service," bye changing this from `execute` to `create`